kernel/reduction.mli

```diff
@@ -103,6 +103,12 @@ val beta_app : constr -> constr -> constr
 (** Pseudo-reduction rule  Prod(x,A,B) a --> B[x\a] *)
 val hnf_prod_applist : env -> types -> constr list -> types
 
+(** In [hnf_prod_applist_assum n c args], [c] is supposed to (whd-)reduce to
+    the form [∀Γ.t] with [Γ] of length [m] and possibly with let-ins; it
```

Comment:
`m` should be `n`.

Reply:
Idem, was a copy-paste of the same typo in a comment of `term.mli`. Fixed here and in `term.mli`